WireDomain/Sources/WireDomain/UseCases/Protocols/ImportBackupError.swift

```diff
@@ -21,7 +21,7 @@ public enum ImportBackupError: Error, Equatable, CaseIterable {
     /// The backup file is encrypted and a password is needed for decryption.
     case passwordRequired
     /// E.g. if the file to import was created with a different (incompatible) version of the app.
-    case incompatibleFileFormat
+    case incompatibleFileFormat // there is no mapping to this error (it's never thrown)
```

Comment:
question: unrelated changes?

Reply:
Yes, it belongs to the fix I mentioned.

Reply:
Moved it to another PR: https://github.com/wireapp/wire-ios/pull/2709